arcade/scene.py

```diff
@@ -396,3 +396,14 @@ def draw_hit_boxes(
 
         for sprite_list in self._sprite_lists:
             sprite_list.draw_hit_boxes(color, line_thickness)
+    
+    def __bool__(self) -> bool:
+        """Returns whether or not _sprite_lists contains anything"""
+        return bool(self._sprite_lists)
+
+    def __contains__(self, item: Union[str, SpriteList]) -> bool:
+        """
+        Returns whether or not the item is in _sprite_lists 
+        or is a value in _name_mapping
+        """
```

Comment:
Minor nitpick: I think you can reduce this to a a one-line docstring if you don't want to add anything additional lines.
```suggestion
"""True when `item` is in `_sprite_lists` or is a value in `_name_mapping`"""
```